staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -162,6 +163,26 @@ func (r *Request) Suffix(segments ...string) *Request {
 	return r
 }
 
+// AsTable adds the "as=Table;v=v1alpha1;g=meta.k8s.io" parameter to the Accept header
+func (r *Request) AsTable(gv metav1.GroupVersion) *Request {
+	if len(gv.Group) == 0 {
+		gv.Group = metav1alpha1.GroupName
+	}
+	if len(gv.Version) == 0 {
+		gv.Version = metav1alpha1.SchemeGroupVersion.Version
+	}
+
+	tableParam := fmt.Sprintf(";as=Table;v=%s;g=%s", gv.Version, gv.Group)
+
+	if len(r.content.AcceptContentTypes) > 0 {
+		r.SetHeader("Accept", r.content.AcceptContentTypes+tableParam)
```

Comment:
Ah, it's not.  It seems like we'll eventually want a way to add or set content types, but this could hold for a while.